index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <div id="main">
+      <title>VictorAhmed</title>
+      <div id="img-div"><img src="https://images.unsplash.com/photo-1551150441-3f3828204ef0?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80"  alt="logo">
+      <p id="img-caption">Horse Logo in black background</p>
+      </div>
+      <p id="tribute-info">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Maiores inventore distinctio nihil ex suscipit. Omnis, officiis distinctio pariatur in excepturi dignissimos! Nobis maxime accusantium neque, ipsa officiis suscipit voluptates reprehenderit, quia eos, inventore sit nam dignissimos. Quo, repudiandae? Harum perferendis reprehenderit, eveniet amet fugit saepe.</p>
+      <a href="alibaba.com" target="_blank" id="tirbute-link">Alibaba</a>
+    </div>
```

Comment:
Please use proper html tag semantics, for example, you could use a section tag in place of that div#main tag. This is the new proper way of using html tags.
Also use proper indentation so as to make your code neat and clean and also for easy accessibility.
Please use a descriptive alt attribute for your logo image. This enables screen readers to enable users with disability understand what the logo represents.